<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>34</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>BOD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>T.H</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Mg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Ca</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>T.D.S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>CL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>SO4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>NO3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>PO4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>EC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>PH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Na</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>101</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="BOD_ob"/>
     </item>
     <item>
      <widget class="QLineEdit" name="TH_ob"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Mg_ob"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Ca_ob"/>
     </item>
     <item>
      <widget class="QLineEdit" name="TDS_ob"/>
     </item>
     <item>
      <widget class="QLineEdit" name="CL_ob"/>
     </item>
     <item>
      <widget class="QLineEdit" name="SO4_ob"/>
     </item>
     <item>
      <widget class="QLineEdit" name="NO3_ob"/>
     </item>
     <item>
      <widget class="QLineEdit" name="PO4_ob"/>
     </item>
     <item>
      <widget class="QLineEdit" name="EC_ob"/>
     </item>
     <item>
      <widget class="QLineEdit" name="PH_ob"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Na_ob"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>101</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="BOD_vstd"/>
     </item>
     <item>
      <widget class="QLineEdit" name="TH_vstd"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Mg_vstd"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Ca_vstd"/>
     </item>
     <item>
      <widget class="QLineEdit" name="TDS_vstd"/>
     </item>
     <item>
      <widget class="QLineEdit" name="CL_vstd"/>
     </item>
     <item>
      <widget class="QLineEdit" name="SO4_vstd"/>
     </item>
     <item>
      <widget class="QLineEdit" name="NO3_vstd"/>
     </item>
     <item>
      <widget class="QLineEdit" name="PO4_vstd"/>
     </item>
     <item>
      <widget class="QLineEdit" name="EC_vstd"/>
     </item>
     <item>
      <widget class="QLineEdit" name="PH_vstd"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Na_vstd"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>101</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLineEdit" name="BOD_uwt"/>
     </item>
     <item>
      <widget class="QLineEdit" name="TH_uwt"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Mg_uwt"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Ca_uwt"/>
     </item>
     <item>
      <widget class="QLineEdit" name="TDS_uwt"/>
     </item>
     <item>
      <widget class="QLineEdit" name="CL_uwt"/>
     </item>
     <item>
      <widget class="QLineEdit" name="SO4_uwt"/>
     </item>
     <item>
      <widget class="QLineEdit" name="NO3_uwt"/>
     </item>
     <item>
      <widget class="QLineEdit" name="PO4_uwt"/>
     </item>
     <item>
      <widget class="QLineEdit" name="EC_uwt"/>
     </item>
     <item>
      <widget class="QLineEdit" name="PH_uwt"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Na_uwt"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>101</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLineEdit" name="BOD_qr"/>
     </item>
     <item>
      <widget class="QLineEdit" name="TH_qr"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Mg_qr"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Ca_qr"/>
     </item>
     <item>
      <widget class="QLineEdit" name="TDS_qr"/>
     </item>
     <item>
      <widget class="QLineEdit" name="CL_qr"/>
     </item>
     <item>
      <widget class="QLineEdit" name="SO4_qr"/>
     </item>
     <item>
      <widget class="QLineEdit" name="NO3_qr"/>
     </item>
     <item>
      <widget class="QLineEdit" name="PO4_qr"/>
     </item>
     <item>
      <widget class="QLineEdit" name="EC_qr"/>
     </item>
     <item>
      <widget class="QLineEdit" name="PH_qr"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Na_qr"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>101</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLineEdit" name="BOD_wnqn"/>
     </item>
     <item>
      <widget class="QLineEdit" name="TH_wnqn"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Mg_wnqn"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Ca_wnqn"/>
     </item>
     <item>
      <widget class="QLineEdit" name="TDS_wnqn"/>
     </item>
     <item>
      <widget class="QLineEdit" name="CL_wnqn"/>
     </item>
     <item>
      <widget class="QLineEdit" name="SO4_wnqn"/>
     </item>
     <item>
      <widget class="QLineEdit" name="NO3_wnqn"/>
     </item>
     <item>
      <widget class="QLineEdit" name="PO4_wnqn"/>
     </item>
     <item>
      <widget class="QLineEdit" name="EC_wnqn"/>
     </item>
     <item>
      <widget class="QLineEdit" name="PH_wnqn"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Na_wnqn"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>        Parameters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Observed Value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>     V Standard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>            Unit Weight</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>              Quality rating                             </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>      WnQn</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="newparam">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>150</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>New Parameter</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>470</y>
      <width>661</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="sum1">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sum2">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sum3">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sum4">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sum5">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionPlot_Graph"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionPlot_Graph">
   <property name="text">
    <string>Plot Graph</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <signal>display(double)</signal>
 </slots>
</ui>
